RestSharp/RestClient.cs

```diff
@@ -390,10 +294,50 @@ public Uri BuildUri(IRestRequest request)
             return new Uri(assembled);
         }
 
+        /// <summary>
+        ///     Retrieve the handler for the specified MIME content type
+        /// </summary>
+        /// <param name="contentType">MIME content type to retrieve</param>
+        /// <returns>IDeserializer instance</returns>
+        private IDeserializer GetHandler(string contentType)
```

Comment:
Is/should this method be tested?

Reply:
May be but this is not something I created.